File: mobile/src/main/java/com/backdoor/moove/core/utils/AssetsUtil.java
package com.backdoor.moove.core.utils;

import android.content.Context;
import android.graphics.Typeface;
import androidx.annotation.NonNull;

public class AssetsUtil {
    public AssetsUtil() {
    }

    @NonNull
    public static Typeface getLightTypeface(Context context) {
        return Typeface.createFromAsset(context.getAssets(), "fonts/Roboto-Light.ttf");
    }

    @NonNull
    public static Typeface getMediumTypeface(Context context) {
        return Typeface.createFromAsset(context.getAssets(), "fonts/Roboto-Medium.ttf");
    }
}

File: mobile/src/main/java/com/backdoor/moove/core/utils/AssetsUtil.kt
package com.backdoor.moove.core.utils

import android.content.Context
import android.graphics.Typeface

class AssetsUtil {
    companion object {

        fun getLightTypeface(context: Context): Typeface {
            return Typeface.createFromAsset(context.assets, "fonts/Roboto-Light.ttf")
        }

        fun getMediumTypeface(context: Context): Typeface {
            return Typeface.createFromAsset(context.assets, "fonts/Roboto-Medium.ttf")
        }
    }
}
